pkg/component/etcd/serviceaccount/serviceaccount_suite_test.go

```diff
@@ -21,7 +21,7 @@ import (
 	. "github.com/onsi/gomega"
 )
 
-func TestService(t *testing.T) {
+func TestServiceAccount(t *testing.T) {
```

Comment:
see the above comment. This should not be included as part of this PR.

Reply:
Thanks Madhav.
Reverted the changes